Developing-new-features-with-feature-gating.md

```diff
@@ -230,12 +315,14 @@ Say you are working on a large scale user-facing feature that will take multiple
 
 3. The very last PR you make (to finish up the feature you are working on) must include changes that move the feature flag to the TEST stage. This is to ensure that the feature is available in the test environment, and we can feature-test it before it is made available to the users in the production environment. **NOTE: Please test all the changes manually to make sure that the feature works fully end-to-end on your local dev server, before flipping the flag to TEST.**
 
-4. If the feature testing reveals that the feature is not ready for production, you must work on fixing the highlighted issues before proceeding further.
+4. Ask the release testers to enable the feature flag of the feature you are building for the TEST environment (in the test server, of course), before proceeding with the actual testing.
```

Comment:
Don't need the "of course" :) Also the release testers in general won't be able to manipulate feature flags. I suggest:

"Ask the release coordinator to go to the test server admin page and enable the feature flag for the feature you are building, before opening up the server for the actual testing."

Reply:
Done!